crates/autopilot/src/driver_model.rs

```diff
@@ -135,20 +135,34 @@ pub mod solve {
 
     #[derive(Clone, Debug, Default, Deserialize)]
     #[serde(deny_unknown_fields)]
-    pub struct Response {
+    pub struct Solution {
+        pub id: u64,
```

Comment:
nit: can you please add a comment that this is a solution id local to the solver (to identify solutions later on when revealing them). When I first read this struct I thought this would be the id of the solve request (ie the auction id)